include/bluetooth/services/ots_client.h

```diff
@@ -1,28 +1,51 @@
 /* @file
- * @brief Object Transfer client header
+ * @brief Object Transfer Service client header
```

Comment:
```suggestion
 * @brief Object Transfer Service Client header
```

Reply:
@Thalley Do we have a tradition on this?  I capitalized the OTS, as that is the name of the service.

Reply:
We don't, and `client` is more prevailent than `Client`. The only place where `Client` is used is actually for the previously named `OTC`, so I understand why this comment was made :)

Reply:
Well, now the file is removed anyway.